pkg/gds/emails/client.go

```diff
@@ -485,3 +486,40 @@ func (m *EmailManager) SendReissuanceStarted(vasp *pb.VASP, whisperLink string)
 
 	return sent, errs.ErrorOrNil()
 }
+
+// SendReissuanceAdminNotification sends the admins a notice that an identity certificate
+// has been reissued. This allows the admins that the reissuance has been done automatically
+func (m *EmailManager) SendReissuanceAdminNotification(vasp *pb.VASP, reissueDate time.Time) (sent int, err error) {
+	// Create the template context
+	ctx := ReissuanceAdminNotificationData{
+		VID:                 vasp.Id,
+		CommonName:          vasp.CommonName,
+		Endpoint:            vasp.TrisaEndpoint,
+		RegisteredDirectory: m.conf.DirectoryID,
+		Reissuance:          reissueDate,
+		BaseURL:             m.conf.AdminReviewBaseURL,
+	}
+	// TODO: check if email has been sent to trisa admin regarding this specific vasp
+
+	if vasp.IdentityCertificate != nil {
+		// TODO: ensure the timestamp format is correct
+		ctx.SerialNumber = strings.ToUpper(hex.EncodeToString(vasp.IdentityCertificate.SerialNumber))
```

Comment:
Thanks for encoding the serial number properly so that it matches the Sectigo certificate format. I'm wondering if we should also return an error is `vasp.IdentityCertificate` is nil, that would indicate that the certificate was not issued properly and we could return something like `fmt.Errorf("no identity certificate for vasp %s", vasp.Id)`.

Reply:
made this change in latest commit https://github.com/trisacrypto/directory/pull/723/commits/66f9d7eb7e76d223e0b92500f2f3e5cdb769b5c0